src/components/forms/power-vs/PowerVsCloudConnectionForm.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react";
+import {
+  buildFormDefaultInputMethods,
+  buildFormFunctions,
+} from "../../component-utils";
+import { IcseNumberSelect, IcseSelect } from "../../Dropdowns";
+import { IcseMultiSelect } from "../../MultiSelects";
+import { IcseTextInput, IcseNameInput, IcseToggle } from "../../Inputs";
+import { IcseFormGroup } from "../../Utils";
+import PropTypes from "prop-types";
+
+class PowerVsCloudConnectionForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { ...this.props.data };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleToggle = this.handleToggle.bind(this);
+    this.handleMultiSelectChange = this.handleMultiSelectChange.bind(this);
+    buildFormDefaultInputMethods(this);
+    buildFormFunctions(this);
+  }
+
+  handleInputChange(event) {
+    this.setState(this.eventTargetToNameAndValue(event));
+  }
+
+  handleToggle(name) {
+    this.setState(this.toggleStateBoolean(name, this.state));
+  }
+
+  /**
+   * handle multiselect change
+   * @param {string} name key to change in the instance
+   * @param {*} value value
+   */
+  handleMultiSelectChange(name, value) {
+    this.setState(this.setNameToValue(name, value));
+  }
+
+  render() {
+    return (
```

Comment:
let's move _Enable Global Routing_ up onto the first form group. 

Then move Transit Gateways up into the second form group on the far right. 

<img width="871" alt="Screenshot 2023-09-07 at 10 43 20 AM" src="https://github.com/IBM/icse-react-assets/assets/11401353/95b7b136-761b-45a9-a2d9-2af5395aa0d4">